entities/regions.py

```diff
@@ -0,0 +1,42 @@
+from random import randint
+from random import choice
+from config import Config
+
+
+class Regions(object):
```

Comment:
`Region`, singular? It's instantiated with an id, so this is just a region right? Or a subregion (functionally equivalent?).